tools/pepsi/src/logs.rs

```diff
@@ -63,6 +69,17 @@ pub async fn logs(arguments: Arguments) -> Result<()> {
             })
             .await
         }
+        Arguments::List { naos } => {
+            ProgressIndicator::map_tasks(
+                naos,
+                "Retrieving logs...",
```

Comment:
Now `pepsi logs show` and `pepsi logs list` have the same progress message. What do you think about "Retrieving latest logs..." for the `pepsi logs show` and "Retrieving all logs..." for the `pepsi logs list` command?

Reply:
Done!